### PR TITLE
Add grpcio-tools

### PR DESCRIPTION
**Pull request by martinRenou**
Signed-off-by: martinRenou <martin.renou@gmail.com>

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble be please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by martinRenou** [comment]
They don't ship a License file in the tgz file

**Comment by martinRenou** [comment]
Actually, this recipe is wrong. We should compile it.

File: recipes/grpcio-tools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+
+  host:
+    - grpcio >=1.33.2
```
Comment (wolfv) [inline recipes/grpcio-tools/meta.yaml:23]: should this maybe pin to the same version as grpcio-tools? should we use `=={{ version }}` here?

File: recipes/grpcio-tools/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    - protobuf >=3.6.0
+    - python  >=3.6
+
+test:
+  imports:
+    - grpc_tools
+
+about:
+  home: https://grpc.io
+  license: Apache Software
```
Comment (wolfv) [inline recipes/grpcio-tools/meta.yaml:38]: I think the correct short spdx identifier is Apache-2.0 or Apache-1.0

Comment (wolfv) [reply recipes/grpcio-tools/meta.yaml:38]: https://spdx.org/licenses/

**Review by wolfv — CHANGES_REQUESTED** [review]

File: recipes/grpcio-tools/meta.yaml (hunk 1 above)
Comment (martinRenou) [reply recipes/grpcio-tools/meta.yaml:23]: Yes, I thought about it but kept the same as in the Anaconda recipe. Let's do that!

**Pull request merged**